src/DnsHelper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Yiisoft\NetworkUtilities;
+
+class DnsHelper
+{
+    /**
+     * @param string $hostname hostname without dot at end
+     */
+    public static function checkMx(string $hostname): bool
+    {
+        $hostname .= '.';
+        try {
+            if (!@dns_check_record($hostname, 'MX')) {
+                return false;
+            }
+            $result = @dns_get_record($hostname, DNS_MX);
```

Comment:
Won't `dns_check_record` already check for existence? Why do we need `dns_get_record`?

Reply:
Same question about cases above.

Reply:
corrected